### PR TITLE
Battery Condition

### PR DESCRIPTION
**Pull request by Shuzhengz**
Checks current battery condition in percent of original capacity

### CODE REVIEW

File: src/system.rs [hunk 1]
```diff
@@ -181,13 +181,41 @@ fn interpret_turbo(is_turbo: &mut String) -> Result<bool, Error> {
...
+    let mut capacity_readings: String = String::new();
+    let bat_energy_full_path: &str = "/sys/class/power_supply/BAT0/";
+    if design == true {
+        File::open(bat_energy_full_path.to_string() + "energy_full_design")?
+            .read_to_string(&mut capacity_readings)?;
+    } else {
+        File::open(bat_energy_full_path.to_string() + "energy_full")?
+            .read_to_string(&mut capacity_readings)?;
+    }
+    Ok(capacity_readings.parse::<i32>().unwrap())
```
Comment (Shuzhengz) [inline src/system.rs:194]: hmm im getting a `thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ParseIntError { kind: InvalidDigit }'` here

Comment (JakeRoggenbuck) [reply src/system.rs:194]: capacity_readings might have a newline and then cannot be interpreted as a int

Comment (JakeRoggenbuck) [reply src/system.rs:194]: you can just capacity_readings.pop() it

**Comment by JakeRoggenbuck** [comment]
This looks great!

File: src/system.rs (hunk 1 above)
Comment (Shuzhengz) [reply src/system.rs:194]: oooo okay thanks

**Comment by Camerooooon** [comment]
I love this idea!!!

**Comment by JakeRoggenbuck** [comment]
Laptop tested

**Review by JakeRoggenbuck — APPROVED** [review]

**Pull request merged**